internal/resources/k6/resource_project_allowed_load_zones.go

```diff
@@ -0,0 +1,233 @@
+	AllowedLoadZones types.List  `tfsdk:"allowed_load_zones"`
+}
+
+// projectAllowedLoadZonesResource is the resource implementation.
+type projectAllowedLoadZonesResource struct {
+	basePluginFrameworkResource
+}
+
+// Metadata returns the resource type name.
+func (r *projectAllowedLoadZonesResource) Metadata(_ context.Context, _ resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = resourceProjectAllowedLoadZonesName
+}
+
+// Schema defines the schema for the resource.
+func (r *projectAllowedLoadZonesResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description: "Manages allowed load zones for a k6 project.",
+		Attributes: map[string]schema.Attribute{
+			"project_id": schema.Int32Attribute{
+				Description: "The identifier of the project to manage allowed load zones for.",
+				Required:    true,
+			},
+			"allowed_load_zones": schema.ListAttribute{
+				Description: "List of allowed k6 load zone IDs for this project.",
+				Required:    true,
+				ElementType: types.StringType,
+			},
+		},
+	}
+}
+
+// Create creates the resource and sets the Terraform state on success.
+func (r *projectAllowedLoadZonesResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	// Retrieve values from plan
+	var plan projectAllowedLoadZonesResourceModel
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	projectID := plan.ProjectID.ValueInt32()
+
+	// Get load zones from plan
+	var loadZones []string
+	diags = plan.AllowedLoadZones.ElementsAs(ctx, &loadZones, false)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Set allowed load zones
+	err := setProjectAllowedLoadZones(ctx, r.client, r.config, projectID, loadZones)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error setting allowed load zones",
+			"Could not set allowed load zones for k6 project: "+err.Error(),
+		)
+		return
+	}
+
+	// Set state to fully populated data
+	diags = resp.State.Set(ctx, plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+// Read retrieves the resource information.
+func (r *projectAllowedLoadZonesResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	// Get current state
+	var state projectAllowedLoadZonesResourceModel
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	projectID := state.ProjectID.ValueInt32()
+
+	// Verify project exists
+	ctx = context.WithValue(ctx, k6.ContextAccessToken, r.config.Token)
+	_, httpResp, err := r.client.ProjectsAPI.ProjectsRetrieve(ctx, projectID).
+		XStackId(r.config.StackID).
+		Execute()
+
+	if httpResp != nil && httpResp.StatusCode == http.StatusNotFound {
+		resp.State.RemoveResource(ctx)
+		return
+	}
+
```

Comment:
The 404 error in this case means the related project does not exist, which is a configuration error rather than a state diff. It shouldn't be handled gracefully.

```suggestion
```